examples/webgpu_compute_audio.html

```diff
@@ -92,16 +92,18 @@
 
 				sampleRate = audioBuffer.sampleRate / audioBuffer.numberOfChannels;
 
-
 				// create webgpu buffers
 
-				waveGPUBuffer = new THREE.StorageInstancedBufferAttribute( waveBuffer, 1 );
-
-				const waveStorageNode = storage( waveGPUBuffer, 'float', waveBuffer.length );
+				waveArray = instancedArray( waveBuffer );
 
 				// read-only buffer
 
-				const waveNode = storageObject( new THREE.StorageInstancedBufferAttribute( waveBuffer, 1 ), 'float', waveBuffer.length ).toReadOnly();
+				const waveNode = instancedArray( waveBuffer );
+
+				// The Pixel Buffer Object (PBO) is required to get the GPU computed data to the CPU in the WebGL2 fallback.
+				// As used in `renderer.getArrayBufferAsync( waveArray.value )`.
+
+				waveNode.setPBO( true );
```

Comment:
@sunag The example throws a runtime error right now since `setPBO()` does not exist:

> Uncaught (in promise) TypeError: waveNode.setPBO is not a function
    at init (webgpu_compute_audio.html:107:14)

Reply:
Commented out the line for now: https://github.com/mrdoob/three.js/pull/29972

The effect in WebGL 2 does not sound correct though.